src/net_processing.cpp

```diff
@@ -1232,7 +1232,7 @@ void PeerLogicValidation::UpdatedBlockTip(const CBlockIndex *pindexNew, const CB
  * Handle invalid block rejection and consequent peer banning, maintain which
  * peers announce compact blocks.
```

Comment:
Suggested enhancement to comment:

"This is called after cursory DoS checks fail, and after full validation of the block in question."